src/api/forms/service.js

```diff
@@ -133,6 +133,9 @@ export async function updateDraftFormDefinition(formId, definition, author) {
       throw Boom.badRequest(`Form with ID '${formId}' has no draft state`)
     }
 
+    // patch the form definition with attributes from the title, where they're not designed to be customised
```

Comment:
Mind if we explain why it's missing?

Might not need to patch it one day

```suggestion
    // Patch definition name using form title (e.g. from JSON uploads)
```

Reply:
Done